de.metas.business.rest-api/src/main/java/de/metas/rest_api/invoicecandidates/response/JsonInvoiceStatus.java

```diff
@@ -42,4 +42,6 @@
 
 	LocalDate dateInvoiced;
 
+	boolean isPDFAvailable ;
```

Comment:
pls try if you can use @JsonProperty("pdfAvailable") to make the property look good in the rendered JSON output